spec/models/foodstuff_spec.rb

```diff
@@ -31,51 +30,28 @@
           let(:quantity) { " 1/2　" }
           it { is_expected.to eq "1/2" }
         end
-      end
-
-      # begin テストがパスしない
-      context "NG" do
         context "/、数字、スペース以外の文字列を含むこと" do
           let(:quantity) { "aaa" }
-          it { is_expected.to be_falsey }
+          it { is_expected.to "aaa" }
         end
       end
-      # end テストがパスしない
     end
   end
 
-  describe "quantityカラムのバリデーション" do
-    # context "OK" do
-    #   it "数字のみ(半角)であること" do
-    #     foodstuff = build(:foodstuff, :only_half_size_number)
-    #     foodstuff.valid?
-    #     expect(foodstuff).to be_valid
-    #   end
-
-    #   it "数字のみ(全角)であること" do
-    #     foodstuff = build(:foodstuff, :only_full_width_number)
-    #     foodstuff.valid?
-    #     expect(foodstuff).to be_valid
-    #   end
-
-    #   it "分数であること" do
-    #     foodstuff = build(:foodstuff, :only_full_width_number)
-    #     foodstuff.valid?
-    #     expect(foodstuff).to be_valid
-    #   end
-
-    #   it "スペースを含んでいる数字であること" do
-    #     foodstuff = build(:foodstuff, :only_half_size_number_with_space)
-    #     foodstuff.valid?
-    #     expect(foodstuff).to be_valid
-    #   end
+  describe "quantityカラム" do
```

Comment:
- describeの名前の付け方が微妙な気がする - quantityカラム

Reply:
こんな感じでいいですけどね。あえていうなら。

```suggestion
  describe "quantity" do
```

だけでいんじゃないかくらい。